website/docs/r/logging_billing_account_sink.html.markdown

```diff
@@ -13,7 +13,9 @@ Manages a billing account logging sink. For more information see
 [Exporting Logs in the API](https://cloud.google.com/logging/docs/api/tasks/exporting-logs).
 
 Note that you must have the "Logs Configuration Writer" IAM role (`roles/logging.configWriter`)
-granted to the credentials used with terraform.
+[granted on the billing account](https://cloud.google.com/billing/reference/rest/v1/billingAccounts/getIamPolicy) to
```

Comment:
Grammar change: what do you think about moving the subject around to make this sentence less passive? Something along the lines of:

"The credentials you are using with Terraform must have the "Logs Configuration Writer" IAM role (`roles/logging.configWriter`) [granted on the billing account](https://cloud.google.com/billing/reference/rest/v1/billingAccounts/getIamPolicy) that is used with this sink.